### PR TITLE
add ops-guide content

### PR DESCRIPTION
**Pull request by sunhao2017**
add ops-guide content

### CODE REVIEW

File: ops-guide.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# TiDB 运维技术文档
+## 目录
+
++ [软硬件环境需求]()
```
Comment (shenli) [inline ops-guide.md]: 如果没有链接，还不如先不写[]() 否则点进去是 404

**Comment by queenypingcap** [comment]
@sunhao2017 @shenli @zhengwanbo PTAL

**Comment by andelf** [comment]
Should never use hand-written TOC.

How will this repo arrange all documents. Any plan?

**Comment by queenypingcap** [comment]
We don't need this document because the doc structure has included the operations guide. Hence closing the PR.

**Pull request closed without merging**